### ISSUE TITLE
Security Group - Inner_access changing state before apply

### ISSUE THREAD

**Issue by GabrielZabielski** [terraform-providers/terraform-provider-alicloud#1638]
Hi there,

I have found a risky bug with the resource `security group` - `inner_access` is changing the status into `true` before apply. The plan is showing properly parameter state, however, as you can see below, the state of this parameter is changing before apply.

#### [Edit] 
- Okey, so another problem with `deprecated` `inner_access` is the information about it in terraform - it is not visible when you provisioning infrastructure with a lot of module layers. 
Warning information prints out only with simple infrastructure, without modularised security groups. I notice this warning when I create the example below.
PS. I like this new parameter :)

### Terraform Version
Terraform v0.12.8
+ provider.alicloud v1.55.3 <- it's working correctly with 1.55.2

### Affected Resource
- resource.alicloud_security_group

### Terraform Configuration Files
```hcl
resource "alicloud_security_group" "v1_55_3" {
  provider = alicloud.newst
  inner_access = false
}

provider "alicloud" {
  alias = "newst"
  version = "<= 1.55.3"
  profile = "main"
}

output "v3" {
  value = alicloud_security_group.v1_55_3.inner_access
}
```

### Steps to Reproduce
1. `terraform init`
2. `terraform apply`
Output:
```
Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

Outputs:

v3 = true
```

### Important Factoids
It works correctly with 1.55.2 provider version. 
The next `terraform apply` on this module, will change the state of the `inner_access` into correct.

**Comment by xiaozhu36** [comment]
HI @GabrielZabielski This issue has been fixed by 1.55.4 and please check it.

**Comment by GabrielZabielski** [comment]
Sorry for the late - it's working correctly now. 

Thanks guys!
